.github/workflows/trunk-upgrade.yaml

```diff
@@ -0,0 +1,25 @@
+name: Weekly Trunk Upgrade
+on:
+  schedule:
+    # Every Monday @ 5am
+    - cron: 0 5 * * 1
+    # Allows us to manually run the workflow from Actions UI
+  workflow_dispatch: {}
+permissions: read-all
+jobs:
+  trunk_upgrade:
+    name: Upgrade Trunk
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # For trunk to create PRs
+      pull-requests: write # For trunk to create PRs
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Trunk Upgrade
+        uses: trunk-io/trunk-action/upgrade@v1
+        with:
+          base: main
+          reviewers: "@masterpointio/Masterpoint Internal"
```

Comment:
Can't tell if this will work since there's a space, but it's also a string. If this doesn't work, then `masterpoint-internal` will (got this from the API response)
<img width="545" alt="image" src="https://github.com/masterpointio/terraform-aws-tailscale/assets/35899715/032a178a-f6b8-40ec-8632-e9c2a3d9a626">